storage/cloudspanner/log_storage.go

```diff
@@ -572,6 +578,9 @@ func (tx *logTX) DequeueLeaves(ctx context.Context, limit int, cutoff time.Time)
 			break
 		}
 	}
+	if drain > 0 {
+		glog.Warningf("Drained %d", drain)
```

Comment:
Is this bad? Can you expand on the message if necessary.

Reply:
removed